File: java/Cell.java
public class Cell{
   private int row;
   private int col;
   private boolean alive;
   private int neighbours = -1;
   
   public Cell(int r, int c){
      row = r;
      col = c;
      alive = false;
   }
   public Cell(int r, int c, boolean a){
      row = r;
      col = c;
      alive = a;
   }
   public int getRow(){
      return row;
   }
   public int getCol(){
      return col;
   }
   public boolean getAlive()
   {
      return alive;
   }
   public String toString()
   {
      if(alive){
         return "1";
      }
      else{
         return "0";
      }
   }
	public void kill()
	{
		alive = false;
	}
	public void revive()
	{
		alive = true;
	}
	public void setNeighbours(int n)
	{
		neighbours = n;
	}
	public int getNeighbours()
	{
		return neighbours;
	}
}

File: python/game-of-life-2.py/Cell.py
from Board import *

class Cell(object):
	def __init__(self, r, c, a = False):
		self.row = r
		self.col = c
		self.alive = a
		self.neighbours = -1
	def __repr__(self):
		if self.alive:
			return str(1)
		else:
			return str(0)

#cell class is done.
